notification-portlet-webcomponents/notification-icon/src/components/NotificationIcon.vue

```diff
@@ -1,11 +1,11 @@
 <template>
   <div class="notification-icon">
-    <dropdown id="_uid" no-caret :variant="variant">
+    <dropdown id="_uid" no-caret toggle-class="btn-icon">
       <template slot="button-content">
-        <font-awesome-icon icon="bell"/>
-        <span class="count">{{ count }}</span>
+        <font-awesome-icon icon="bell" size="2x" fixed-width/>
+        <span class="count" :class="{ 'alert': count > 1, 'hidden': count <=0 }">{{ countDisplay }}</span>
```

Comment:
:question: `'alert': count > 1` two notifications is when this becomes significant? Should this be `'alert': count > 0`?
:art: : `'hidden': count <=0` I have a mild stylist preference for using `<` over `<=`,  `'hidden': count < 1`